packages/camera/camera/lib/src/camera_preview.dart

```diff
@@ -21,7 +21,7 @@ class CameraPreview extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return controller.value.isInitialized
+    return (controller.value.isInitialized && !controller.value.isPreviewPaused)
```

Comment:
We use camera package intensively and there was error on closing widgets with mounted camera preview.
Actually `controller.value.isInitialized` is not enough, because `const CameraValue.uninitialized()` is never called. And `isInitialized` remains `true` even on disposed controller.
In scenarios with controller disposed in `didChangeAppLifecycleState`: the `isPreviewPaused` is the only flag giving a clue to stop previewing and even listening to controller.value changes is not necessary if preview is not (already not) required.

Reply:
If I understand correctly, you added this to check if the controller is disposed. It is probably better to add a `isDisposed` method to the `CameraController` instead. Although I'm not entirely sure why we added a `debugCheckIsDisposed` instead.